src/main/java/com/rackspace/salus/resource_management/web/controller/ResourceApi.java

```diff
@@ -115,4 +130,16 @@ public void delete(@PathVariable String tenantId,
                        @PathVariable String resourceId) {
         resourceManagement.removeResource(tenantId, resourceId);
     }
+
+    @GetMapping("/tenant/{tenantId}/resourceLabels")
+    public List<Resource> getResourcesWithLabels(@PathVariable String tenantId,
+                                                 @RequestBody Map<String, String> labels) {
```

Comment:
```suggestion
                                                 @RequestParam Map<String, String> labels) {
```

I just noticed I had to make this change locally for the public API proxy to work without customizations -- GETs don't normally have a body.